src/Database/Mysql/Driver/Mysql.php

```diff
@@ -43,11 +43,19 @@ public function create(Tenant $tenant): bool
 
         event(new Events\Creating($tenant, $config, $this));
 
-        return $this->processAndDispatch(Events\Created::class, $tenant, [
-            'user'     => "CREATE USER IF NOT EXISTS `{$config['username']}`@'{$config['host']}' IDENTIFIED BY '{$config['password']}'",
-            'database' => "CREATE DATABASE `{$config['database']}`",
-            'grant'    => "GRANT ALL ON `{$config['database']}`.* TO `{$config['username']}`@'{$config['host']}'",
-        ]);
+        $qm = app(QueryManager::class);
```

Comment:
If you're going to use this class in every public method, please stick it in the `__construct`

Reply:
yeah, I meant to do that before submitting and forgot... whoops.  I'll get that moved over shortly